### PR TITLE
Implementation of issue #166 suggestions.

### PR DESCRIPTION
**Pull request by nerun**
Nothing special here, just a few modifications to make it easier for newbies. Instructions became more didactic and, mainly, updated in relation to the use of the program in its current state.

Issue #166 can be closed now as it has been fully implemented.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -28,37 +28,64 @@ git clone https://github.com/linappleii/linapple.git
 cd linapple
 make
 ```
-For a faster compilation, you can add an option "-jx", where "x" is the number of threads of your CPU. For example, AMD Ryzen 1600 has 6 cores, but 12 threads:
+For a faster compilation, you can add an option "-jX", where "X" is the number of threads of your CPU. For example, *AMD Ryzen 5 2600* has 6 cores, but 12 threads:
 ```bash
 make -j12
 ```
 
-### Global Install
```
Comment (maxolasersquad) [inline INSTALL.md:36]: Why remove this section?

File: INSTALL.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 ### Prerequisites
 
-#### Ubuntu
+#### Debian / Ubuntu
 
 ```bash
 sudo apt-get install git libzip-dev libsdl1.2-dev libsdl-image1.2-dev libcurl4-openssl-dev zlib1g-dev
```
Comment (maxolasersquad) [inline INSTALL.md:9]: Can you append `imagemagick` to this list? I think Fedora needs `GraphicsMagick`, but I don't have an install readily available, so not sure.

File: INSTALL.md (hunk 1 above)
Comment (nerun) [reply INSTALL.md:36]: Not removed, but moved down.

Comment (maxolasersquad) [reply INSTALL.md:36]: Ah I missed that. :)

File: INSTALL.md (hunk 2 above)
Comment (nerun) [reply INSTALL.md:9]: For sure! Working on it.

**About Fedora**: i don't use it too, it's legacy install instructions, but seems you are right, [read here](https://www.tecmint.com/install-imagemagick-in-linux/).

**Review by maxolasersquad — APPROVED** [review]

File: INSTALL.md (hunk 2 above)
Comment (nerun) [reply INSTALL.md]: Done! You was right about Fedora!

**Pull request merged**